src/pfb/base.py

```diff
@@ -56,7 +56,7 @@ def encode_enum(enumValue):
 
 
 def decode_enum(enumValue):
-    decodedValue = re.sub("_[a-z0-9]+_", unicode_decode, enumValue, flags=re.UNICODE)
+    decodedValue = re.sub("_[a-f0-9]{2,}_", unicode_decode, enumValue, flags=re.UNICODE)
```

Comment:
And what is it `{2,}`?

Reply:
So the problem we were encountering was that there was an enumerated value that was something like `symptom_a_` So to make sure that we were only capturing unicode symbols it should be of length 2 or more. `{2,}` is the regex match for 2 or more characters